### PR TITLE
Accept an unwrapped tag as an attribute value

### PR DESCRIPTION
**Pull request by sophiebits**
Like `<LeftRight left=<a /> right=<b /> />`.

### CODE REVIEW

**Comment by sophiebits** [comment]
(cc @sebmarkbage)

**Comment by sebmarkbage** [comment]
@jeffmo

File: test/fbtest.js [hunk 1]
```diff
@@ -1427,6 +1427,315 @@ var fbTestFixture = {
                     column: 41
                 }
             }
+        },
+
+        '<LeftRight left=<a /> right=<b>monkeys /> gorillas</b> />': {
```
Comment (jeffmo) [inline test/fbtest.js:1432]: Oh man...
Great test case, but let's hope I never have to read code that does this

**Comment by jeffmo** [comment]
Also we'll need to update the transform to support this too, right?

**Comment by sophiebits** [comment]
Maaybe. :)

**Pull request merged**